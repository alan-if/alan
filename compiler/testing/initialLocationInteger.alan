The x Isa object 4
End The x.

The l Isa location
End The l.

Start At l.
