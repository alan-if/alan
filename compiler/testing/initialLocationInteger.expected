########## initialLocationInteger ##########


initialLocationInteger.alan

    1.  The x Isa object 4
=====>                   1

  *1*   101 E : Syntax error. Inserting "at" before this token.
  *1*   355 E : Initial location can only refer to instances inheriting from
                'location' using AT or to instances having the container
                property using IN.


        2 error(s).
        No detected warnings.
        1 informational message(s).

